index.html

```diff
@@ -17,11 +17,56 @@
   </head>
   <body>
     <header>
-      <div class="container"></div>
+      <div class="container">
+        <div class="column-full padding-20">
+          <h1>Code Journal</h1>
+        </div>
+      </div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
-      <div data-view="entries" class="hidden"></div>
+      <div data-view="entry-form">
+        <div data-view="entries" class="hidden"></div>
+        <div class="column-full row">
+          <h1>New Entry</h1>
+        </div>
+        <form id="entry-form">
+          <div class="row">
+            <img
+              id="url-photo"
+              class="column-half"
+              src="/images/placeholder-image-square.jpg" />
+
```

Comment:
I believe your image is too large which is causing your notes section to be too long and adding scroll onto your desktop view. If you apply width directly to an image it will attempt to side based on the image, we want to size based on the `row`